keras/layers/merge.py

```diff
@@ -187,6 +187,25 @@ class Add(_Merge):
     It takes as input a list of tensors,
     all of the same shape, and returns
     a single tensor (also of the same shape).
+
+    # Examples
+
+    ```python
+        from keras.layers.merge import Add  # a layer
+        from keras.layers.merge import add  # a functional interface
+        from keras.layers import Input, Dense
+        from keras.models import Model
```

Comment:
Hm.. okay, but is it mentioned in anyplace? I actually tried just all the stuff I can think of. My opinion is, for this purpose, examples would be better with import lines. What do you think? I'm pretty sure people have been importing layers in very different ways including some undersired ways cuz they don't know.